Formula/php@7.4.rb

```diff
@@ -6,7 +6,7 @@ class PhpAT74 < Formula
   mirror "https://fossies.org/linux/www/php-7.4.26.tar.xz"
   sha256 "e305b3aafdc85fa73a81c53d3ce30578bc94d1633ec376add193a1e85e0f0ef8"
   license "PHP-3.01"
-  revision 1
+  revision 2
```

Comment:
Let's keep the revision bump out, 7.4 will get an update soon enough.

Reply:
All right, reverted.